mbuild/compound.py

```diff
@@ -1153,55 +1153,117 @@ def particles_in_range(
             particle_array = np.array(list(self.particles()))
         return particle_array[idxs]
 
-    def visualize(self, show_ports=False):
-        """Visualize the Compound using nglview.
+    def visualize(self, show_ports=False, use_py3dmol=False):
```

Comment:
Maybe instead of a bool that picks between these two engines, it could be an argument that's passed through?
```suggestion
    def visualize(self, show_ports=False, engine='nglview'):
    """docstring"""
    engine = engine.lower()
    if engine == 'nglview':
        return self._visualize_nglview(show_ports=show_ports)
    elif engine == 'py3dmol':
        return self._visualize_py3dmol(show_ports=show_ports)
    # Future supported viz engines go in new elif lines
    else:
        raise ValueError('Engine {} is not currently supported'.format(engine)')
```